epicbox-hyperskill/go/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
 
 RUN mkdir /checker  \
-    && curl -L -o /checker/kotlin.zip https://github.com/JetBrains/kotlin/releases/download/v1.9.22/kotlin-compiler-1.9.22.zip  \
+    && curl -L -o /checker/kotlin.zip https://github.com/JetBrains/kotlin/releases/download/v2.0.0/kotlin-compiler-2.0.0.zip  \
```

Comment:
2.0.10

https://github.com/JetBrains/kotlin/releases/tag/v2.0.10

Reply:
I know, but last version [atomicfu-jvm](https://github.com/hyperskill/epicbox-images/blob/3a5f8f12460bc6d5bb9620b8b81bdf5b64928763/epicbox-kotlin/Dockerfile#L4) was updated for Kotlin 2.0.0 https://github.com/Kotlin/kotlinx-atomicfu/blob/master/CHANGES.md#version-0250